jecs.luau

```diff
@@ -1181,14 +1176,24 @@ do
 							local on_remove = id_record.hooks.on_remove
 							local to = archetype_traverse_remove(world, id, idr_t_archetype)
 							local empty = #to.types == 0
-							for i = n, 1, -1 do
-								local child = children[i]
-								if on_remove then
+
+							if on_remove then
```

Comment:
I deliberately kept this path a bit more concise considering that it was getting too long before.